app/testSpec/getColorFromPairNumberTest.js

```diff
@@ -0,0 +1,25 @@
+const {MajorColorNames , MinorColorNames, ColorPair} = require( './../colorCoder');

```

Comment:
Good Approach to keep test code separate from production code 👍

Reply:
@prithamvshetty Thank you for your comment , I will adopt this suggestions 👍